lib/tasks/graphs.rake

```diff
@@ -45,11 +45,13 @@ namespace :graphs do
         # primary y-axis
         plot.set("ytics nomirror")
         plot.ylabel("number of users on the calendar day")
+        plot.yrange("[0:]")
 
         # secondary y-axis
         plot.set("y2tics tc lt 2")
         plot.set('y2label "cumulative total number of users" tc lt 2')
         plot.set('format y2 "%.0f"')
+        plot.y2range("[0:]")
```

Comment:
I'm not sure whether setting the `y2` range as well is strictly necessary but it doesn't seem to hurt